docs/upgrading.md

```diff
@@ -163,7 +163,7 @@ yarn add next@11
 
 ### Webpack 5
 
-Webpack 5 is now the default for all Next.js applications. If you did not have custom webpack configuration your application is already using webpack 5. If you do have custom webpack configuration you can refer to the [Next.js webpack 5 documentation](https://nextjs.org/docs/messages/webpack5) for upgrading guidance.
+Webpack 5 is now the default for all Next.js applications. If you did not have a custom webpack configuration your application is already using webpack 5. If you do have a custom webpack configuration you can refer to the [Next.js webpack 5 documentation](https://nextjs.org/docs/messages/webpack5) for upgrading guidance.
```

Comment:
I think "a" is wrong here unless you're talking about "a configuration file". But we're not because configuration can be defined in multiple files.

Reply:
Thanks. Corrected in the latest commit.